### ISSUE TITLE
VSCode 1.17 upgrade: "The package is of bad quality"

### ISSUE THREAD

**Issue by dfarley1** [Microsoft/vscode#35638]
- VSCode Version: 1.17
- OS Version: Linux x64 4.10.0-35-generic

Steps to Reproduce:

1. Download the linux64 .deb package from the official website.
2. Open it in the Ubuntu Software Center.
3. Click "Upgrade"

Trying to update from VSCode 1.16 to 1.17 gives me the following error message:
```
The installation of a package which violates the quality standards isn't allowed. 
This could cause serious problems on your computer. 
Please contact the person or organisation who provided this package file and include 
the details beneath.

Lintian check results for /home/daniel/Downloads/code_1.17.0-1507160143_amd64 (1).deb:
E: code: forbidden-postrm-interpreter #!/usr/bin/env
```
I've installed and updated VSCode using this method several times in the past without issue.  I've tried re-downloading the package and rebooting as well.

**Comment by Obbi89** [comment]
Hi as I stated on my issue it seems the web download and repository file differ.
The web download ships the right file for me, while the repository contains a bogus file that does not work.
Guess the fix for now is to manually download the file.

**Comment by Tyriar** [comment]
Fixed in master: e29c517386fe6f3a40e2f0ff00effae4919406aa

**Comment by Tyriar** [comment]
This was introduced by https://github.com/Microsoft/vscode/pull/34543

**Comment by Tyriar** [comment]
Reverted the change on release/1.17 as well https://github.com/Microsoft/vscode/commit/9c89921a76d59850222d120e3ba7d5d99c761bf8

**Comment by octref** [comment]
I was verifying this, and Ubuntu Software Center doesn't seem to do the updates correctly.

![update](https://user-images.githubusercontent.com/4033249/31401748-a7c19422-ada8-11e7-8f99-e697a0c3e010.gif)

Result of the second update, where I entered password:

![image](https://user-images.githubusercontent.com/4033249/31401733-9cffab8c-ada8-11e7-8bd8-f6d2ac41c1b3.png)

Which is really strange, since I'm sure I was downloading this build: 1e9d36539b0ae51ac09b9d4673ebea4e447e5353

Basically:

- 1st try of "update" in Ubuntu Software Center -> Stays at 1.16.1
- 2nd try of "update" in Ubuntu Software Center -> Asks for Password -> Only updates to 1.17?

**Comment by chrmarti** [comment]
@octref You always need to enter a password when you install if I remember correctly. So I think the first attempt didn't do anything and the second attempt with the password shows 1.16.1 as the version in the Software Center. Maybe you had opened both .debs? Could you retry?

**Comment by octref** [comment]
I can verify that `dpkg` works though. Maybe it's an incompatibility with Ubuntu Software Center.

**Comment by chrmarti** [comment]
That might be dependent on the Software Center's version. I don't see the problem on Ubuntu 14.04. Not the original problem.

**Comment by Tyriar** [comment]
@octref @chrmarti the software center not updating thing is a longstanding issue which AFAIK is a software center problem.
